packages/@react-spectrum/dialog/chromatic/Dialog.chromatic.tsx

```diff
@@ -89,6 +89,7 @@ storiesOf('Dialog', module)
   );
 
 storiesOf('Dialog/Alert', module)
+  .addParameters({chromaticProvider: {colorSchemes: ['light'], locales: ['en-US'], scales: ['medium'], disableAnimations: true}})
```

Comment:
I assumed this line was for setting the `disableAnimations: true`. When I tested locally we don't have to set the rest since they are defaults.

Reply:
I am unsure what the question is?